README.md

```diff
@@ -24,25 +24,49 @@ EluvioLive CLI
 Usage: elv-live <command>
 
 Commands:
```

Comment:
maybe we should reconsider the narrow format of this, like 120 cols instead of 80. 

and/or sections and/or alphabetical. 

can deal w/ it in a separate PR

Reply:
I agree.  It vomits up huge unreadable list with ./elv-live --help

yargs has a bunch of little config.  looks like terminalWidth is configurable, windowWidth or wrapping

Reply:
Can revisit yargs options or arranging with sections or new spacing/width.  For now, split out the cut and paste of '--help' output to a linked file named CMDLINE.md.